docs/api/consumerauthorization/authorization-token-management-generic-http.md

```diff
@@ -0,0 +1,240 @@
+# authorizationTokenManagement IDD
+**generic_http & generic_https**
+
+## Overview
+
+This page describes the generic_http and generic_https service interface of authorizationTokenManagement, which allows systems (with operator role or proper permission) to manage the service access tokens in bulk and on behalf of the consumer and provider systems. Access tokens enable the verification of service consumption permissions on the provider system side, and the application of session-based service consumption control between the consumer and provider systems. An example of this interaction when a Core/Support system generates tokens for a consumer system for multiple service instances. Tokens for Event notification are also handled by this service in an event publisher/subscriber scenario. To enable other systems to use, to consume it, this service needs to be offered through the ServiceRegistry.
+
+The interfaces are implemented using protocol, encoding as stated in the following tables:
+
+## Interface Description
+
+**generic_http**
+
+Profile type | type | Version
+--- | --- | ---
+Transfer protocol | HTTP | 1.1
+Data encryption | N/A | -
+Encoding | JSON | RFC 8259
+Compression | N/A | -
+
+**generic_https**
+
+Profile type | type | Version
+--- | --- | ---
+Transfer protocol | HTTPS | 1.1
+Data encryption | TLS | -
+Encoding | JSON | RFC 8259
+Compression | N/A | -
+
+Hereby the **Interface Design Description** (IDD) is provided to the [authorizationTokenManagement – Service Description](../../assets/sd/5_0_0/authorization-token-management_sd.pdf). For further details about how this service is meant to be used, please consult that document.
+
+### generate-tokens
+
+The service operation **request** requires an [identity related header or certificate](../authentication_policy.md/#http) and an [AuthorizationTokenGenerationListMgmtRequest](../data-models/authorization-token-generation-list-mgmt-request.md) JSON encoded body. The URI can contain an optional query parameter with the key "_unbound_" and a [Boolean](../primitives.md#boolean) value. If `true` the consumers' service permission check will be skipped if the requester system name is present in the _unbounded.token.generation.whitelist_ system cunfiguration. 
```

Comment:
still see cunfiguration in github. Maybe just a rendering problem, but please double check it.